src/cdk/testing/event-objects.ts

```diff
@@ -74,7 +74,7 @@ export function createKeyboardEvent(type: string, keyCode: number, target?: Elem
 }
 
 /** Creates a fake event object with any desired event type. */
-export function createFakeEvent(type: string, canBubble = true, cancelable = true) {
+export function createFakeEvent(type: string, canBubble = false, cancelable = true) {
```

Comment:
I think that we should leave this one as `true`, because most of the events bubble. We can opt-out of it for the ones that don't.

Reply:
We can also check for the type and just default to `false` if the event is either `focus` | `blur`.

Reply:
We could, but there might be more that don't bubble. We open ourselves up to having to maintain a list of events that don't bubble.

Reply:
That's why I think that we should be explicit the other way. If some tests insist on event bubbling, which is very unlikely, then that test should explicitly turn on `bubbling`. 

Bubbling will most likely causing more side-effects, so disabling it by default in tests might be better. That's at least what I noticed while working with the `focus` event right now.